gen_wiki.py

```diff
@@ -56,6 +56,16 @@ def overview(multi_page: bool = False) -> str:
           for i, name in enumerate(["poketes", "attacks", "types", "item",
                                     "effects"])) + "\n"
 
+    @staticmethod
+    def get_name(poke):
```

Comment:
You missed using this method in line `218` when getting the name of the evolve pokete!